src/components/CourseCard/index.tsx

```diff
@@ -52,6 +57,7 @@ const CourseCard = ({ course, enrolls }: Props) => {
         }}
       >
         <CardHeader
+          sx={{ paddingBottom: !course?.image ? '1rem' : 0 }}
```

Comment:
```suggestion
          sx={{ paddingBottom: !course.image ? '1rem' : 0 }}
```
Check not needed as there will be nothing to render anyway unless given a course